docs/stackscli/about.md

```diff
@@ -10,6 +10,33 @@ The Ensono Stacks CLI is built using Go, which means that it is a single binary
 
 The CLI will help you get started with scaffolding your applications and workspaces using Ensono Stacks! Through a series of questions the CLI will determine how and what to build for your workspace, helping to accelerate your development process!
 
+## Installation
+
+As the CLI is a single binary the quickest way to install it is to download it and put it somewhere in your PATH. The following code snippets show how this can be achieved using Bash and PowerShell.
```

Comment:
```suggestion
As the CLI is a single binary, the quickest way to install it is to download it and add it to your PATH. The following code snippets show how this can be achieved using Bash and PowerShell.
```